internal/csi-common/nodeserver-default.go

```diff
@@ -195,3 +195,13 @@ func ConstructMountOptions(mountOptions []string, volCap *csi.VolumeCapability)
 	}
 	return mountOptions
 }
+
+// Contains checks the opt is present in mountOptions
+func Contains(mountOptions []string, opt string) bool {
```

Comment:
This would be better named to MountOptionContains for clarity.

Reply:
done